### PR TITLE
Date Example

### PR DESCRIPTION
**Pull request by NaWer**

### CODE REVIEW

File: eg/examples/date.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    date --rfc-3339=seconds
+    date --utc
+
+
+Print the local time for 9PM next Friday on the west coast of the US
+
+    date --date='TZ="America/Los_Angeles" 09:00 PM next Friday'
+
+
+Convert a date string into timestamp :
```
Comment (scorphus) [inline eg/examples/date.md]: Apart from this *dangling* colon [cont...]

File: eg/examples/date.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    date --date='25 Dec' +%A
+
+
+Set the system clock forward by two minutes
+
+    date --set='+2 minutes'
+
+
+
+A very complete documentation with a lot of examples is avaible at http://www.gnu.org/software/coreutils/date or with the command `info coreutils 'date invocation'`
```
Comment (scorphus) [inline eg/examples/date.md]: [..cont] and from this typo (`s/availble/available/`) this adds a lot to the project. Thanks!

**Comment by srsudar** [comment]
This is a tricky one! It looks like the commands even for basic options vary quite a bit between Linux and OSX. I'm doing some investigating about how we might represent both adequately. I'll try and have a proposed version for you to look at soon.

**Comment by srsudar** [comment]
Sorry again for the delay on this. I've had PhD deadlines since getting back from an internship this summer (northern hemisphere summer, that is), but I promise I'll merge the outstanding PRs soon.

**Pull request merged**